pyteal/compiler/constants.py

```diff
@@ -94,6 +90,20 @@ def extractAddrValue(op: TealOp) -> Union[str, bytes]:
     return value
 
 
+def extractMethodValue(op: TealOp) -> bytes:
+    """Extract the constant value being loaded by a TealOp whose op is Op.method.
+
+    Returns:
+        The bytes of method selector computed from the method signature that the op is loading.
+    """
+    if len(op.args) != 1 or type(op.args[0]) != str:
+        raise TealInternalError("Unexpected args in method opcode: {}".format(op.args))
+
+    methodSignature = cast(str, op.args[0])
```

Comment:
I think need to discard the leading and trailing `"` before hashing

Reply:
The info stored in `Method` is unprocessed, i.e., without leading/tailing `"`: see https://github.com/algorand/pyteal/blob/feature/method-pseudo-op/pyteal/ast/method.py#L33, where we add `"` in compilation.